tests/modules/ai/application/events/format-upcoming-events-message.test.ts

```
@@ -0,0 +1,164 @@
+import { describe, expect, it } from "vitest";
+import {
+	applyUpcomingEventsFocus,
+	formatUpcomingEventsMessage,
+	splitUpcomingEventsByType,
+	type AvailableUpcomingOccasionCounts,
+	type UpcomingEventsFocus,
```

Biome (2.4.9)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)